src/Microsoft.VisualStudio.Composition/ExportDefinitionBinding.cs

```diff
@@ -51,12 +51,22 @@ public MemberInfo ExportingMember
         /// </summary>
         public bool IsStaticExport
         {
-            get { return this.ExportingMember.IsStatic(); }
+            get { return this.ExportingMemberRef.IsStatic(); }
         }
 
         public TypeRef ExportedValueTypeRef
         {
-            get { return TypeRef.Get(this.ExportedValueType, this.PartDefinition.TypeRef.Resolver); }
+            get
+            {
+                if (this.ExportingMemberRef == null)
+                {
+                    return this.PartDefinition.TypeRef;
+                }
+                else
+                {
+                    return ReflectionHelpers.GetExportedValueTypeRef(this.PartDefinition.TypeRef, this.ExportingMemberRef);
+                }
+            }
         }
 
         public Type ExportedValueType
```

Comment:
>ExportedValueType [](start = 20, length = 17)

Can we revise this to be in terms of `ExportedValueTypeRef`? Not only will that reduce code and match the pattern used elsewhere, but it will force all code paths to use your new code which will help prove that it works in all cases.